cpp/harvest/nacjd_data_pub_pipeline.sh

```diff
@@ -139,7 +141,14 @@ echo "Extracting downloaded information"
 ls -1 *.json | xargs -I'{}' sh -c 'echo ${@%%.json}\\t$(cat $@ | jq -r -C .projectTitle)\\t$(cat $@ | jq -r '"'"'.creators | map("\(.orgName), \(.personName)") | join("; ")'"'"')' _ '{}' | sed -re 's/([, ]+)?null([, ]+)?//g' > $OLD_NACJD_MISSING_STUDIES_ID_TITLE_AUTHOR
 
 echo "Adding information about studies, that are not in K10Plus to 787"
-$AUGMENTING_787_TOOL  $NACJD_WITH_MISSING_SOME_STUDY_LINK $OLD_NACJD_MISSING_STUDIES_ID_TITLE_AUTHOR  $NACJD_FINAL
+$AUGMENTING_787_TOOL  $NACJD_WITH_MISSING_SOME_STUDY_LINK $OLD_NACJD_MISSING_STUDIES_ID_TITLE_AUTHOR  $NACJD_STUDIES
 
 echo "List the ISSNs to be considered"
 $NACJD_TOOL "--verbose" "suggested_report" $NOT_FOUND_ISSN $SOURCE $ISSN_TO_BE_CONSIDERED
+
+echo "Update 007 and 856"
+marc_augmentor $NACJD_STUDIES $NACJD_UPDATE_007_856 --replace-field-if '007:cr|||||' '856u:\W.*' --add-subfield-if '787i:Forschungsdaten' '787t:\W+'
+
+echo "Update leader"
```

Comment:
Please explain for s.o. not involved in the project why and under which circumstances this is necessary.